cli/pcluster/pcluster.py

```diff
@@ -707,7 +696,18 @@ def command(args, extra_args):  # noqa: C901 FIXME!!!
 
         # run command
         if not args.dryrun:
-            os.system(cmd)
+            if return_output:
+                try:
+                    return sub.check_output(cmd, shell=True, stderr=sub.STDOUT)
+                except sub.CalledProcessError as e:
+                    error = e.output.decode("utf-8")
+                    if "/opt/parallelcluster/scripts/pcluster_dcv_connect.sh: No such file or directory" in error:
+                        LOGGER.error("The version of parallelcluster in the machine does not support DCV")
```

Comment:
parallelcluster is not the product name. The name of the product is "ParallelCluster" so we should always log the correct name